web_responsive/static/src/js/web_responsive.js

```diff
@@ -527,14 +527,32 @@ odoo.define('web_responsive', function (require) {
     // `KeyboardNavigationMixin` is used upstream
     AbstractWebClient.include(KeyboardNavigationShiftAltMixin);
 
-    // DocumentViewer: Add support to maximize/minimize
+    // DocumentViewer: Add support to maximize/minimize and changes key events
+    // to listen globally
     DocumentViewer.include({
-        events: _.extend(DocumentViewer.prototype.events, {
+        events: _.extend(_.omit(DocumentViewer.prototype.events, [
+            'keydown',
+            'keyup',
+        ]), {
             'click .o_maximize_btn': '_onClickMaximize',
             'click .o_minimize_btn': '_onClickMinimize',
             'shown.bs.modal': '_onShownModal',
         }),
 
+        start: function () {
+            core.bus.on('keydown', this, this._onKeydown);
+            core.bus.on('keyup', this, this._onKeyUp);
+            const res = this._super.apply(this, arguments);
+            console.log(this.$el);
+            return res;
```

Comment:
It seems you left some :wastebasket: here.

Reply:
Oops! removed. Thanks!